contracts/VotingMachines/IntVoteInterface.sol

```diff
@@ -34,26 +34,26 @@ interface IntVoteInterface {
         ) external returns(bytes32);
 
     // Only owned proposals and only the owner:
-    function cancelProposal(bytes32 _proposalId) external onlyProposalOwner(_proposalId) votable(_proposalId) returns(bool);
```

Comment:
What are the use cases that motivate removing `onlyProposalOwner` and `votable` in all these cases?

Reply:
Empty modifiers are ignored .
The compiler was complaining about it .